app/controllers/v1/users/registrations_controller.rb

```diff
@@ -0,0 +1,32 @@
+module V1::Users
+  class RegistrationsController < ApplicationController
+    before_action :authenticate_user!, except: :create
+
+    def create
+      user = User.new(create_params)
+
+      if user.save
+        @current_user = user
```

Comment:
dałem to `@current_user` z myślą że mogę to zaraz użyć, ale jak widać nie jest do niczego potrzebne. Już wyrzucam